firmware/boron-ins-fsm/test/consoleFunctionTests.cpp

```diff
@@ -44,6 +44,14 @@ SCENARIO( "Turn_Debugging_Publishes_On_Off", "[toggle debug flag]" ) {
                 REQUIRE( returnVal == -1 );
             }
         }
+
+        WHEN("The function is called with 'e'") {
+            int returnVal = toggle_debugging_publishes("e");   
+
+            THEN("TThe function should return whether debugging publishes are turned on, in this case, false") {
```

Comment:
Typo: `Tthe`

Reply:
Thanks, I fixed the typo in my latest commit